contributions/catalog/webapps/templates/contribute/home.html

```diff
@@ -264,10 +272,11 @@ <h4 class="talent_name"></h4>
             ajax('GET', get_cap_url, null,
                 function (res) {
                     var items = JSON.parse(res);
-                    all_capability = items
-                    append_capability(all_capability);
-                    filterSelection('all');
+                    all_capability = items;
+                    // append_capability(all_capability);
+                    all_capability.length > 0? append_capability(all_capability): append_capability(test_capability);
```

Comment:
Do we need this test_capability and test_talent mentioned in this file?

Reply:
no, that is for testing purpose when the API not work well. We can remove it

Reply:
Thanks. I figured it was for local debugging. Please remove those.

Reply:
removed